shared/src/test/diff/nu/BadClassInherit.mls

```diff
@@ -25,15 +25,13 @@ class C2 extends C1(y) {
 //│ class C2 extends C1 {
 //│   let y: Int
 //│ }
-//│ Code generation encountered an error:
-//│   unresolved symbol y
```

Comment:
Do you know why this is no longer a code-gen error? What does it generate?

Reply:
It accesses the signature unexpectedly. Now it is fixed.

Reply:
I don't understand your fix. It doesn't cover other similar situations, such as:
```scala
class C2 extends C1(y) {
  val y: Int = 1
}
```

BTW why are you not introducing a new scope for the part of the class between the `{...}`?